interface/templates/interface/upload.html

```diff
@@ -13,6 +13,12 @@
         {% include "interface/tracker.html" %}
     </head>
 
+    {% if delta_t %}
```

Comment:
Ok, so the `if` makes sense, but it's not clear what the condition means. Let's write it as `{% if too_many_submissions %}` or something.